### PR TITLE
feat: add `component.manufacture`

### LINKED ISSUE #346: Should `manufacture` be a property of `component`, rather than `metadata`?
The majority of fields under `metadata` refer to the BOM itself. 

However, `metadata.manufacture` technically refers to `metadata.component`:

https://github.com/CycloneDX/specification/blob/299209abd9531d808e0cc4235e77a7c4b1b53d96/schema/bom-1.5.schema.json#L263-L267

This can cause confusion because, as mentioned, everything else refers to the BOM, not `metadata.component`.

Should `manufacture` be a property of `component` instead, to make the association more clear?

### PR DESCRIPTION
**Pull request by jkowalleck**
fixes #346 

- [x] json schema
- [x] xml schema
- [x] protobuff schema
- [x] examples

### CODE REVIEW

File: schema/bom-1.6.schema.json [hunk 1]
```diff
@@ -678,8 +678,8 @@
           "$ref": "#/definitions/component"
         },
         "manufacture": {
-          "title": "Manufacture",
-          "description": "The organization that manufactured the component that the BOM describes.",
+          "title": "Manufacturer",
+          "description": "The organization that manufactured the CycloneDX document (the \"manufacturer\", although the property is misspelled).\nThis may be different from the manufacturer of the component that the CycloneDX document describes.",
```
Comment (jkowalleck) [inline schema/bom-1.6.schema.json:682]: :exclamation: this would be a semantic change. technically this is a breaking change.

**Comment by stevespringett** [comment]
Since this is a new field of component, should we use `manufacturer` instead of `manufacture`? I'd like to eventually change `metadata. manufacture` in v2.0 of the spec.

**Comment by jkowalleck** [comment]
> Since this is a new field of component, should we use `manufacturer` instead of `manufacture`? I'd like to eventually change `metadata. manufacture` in v2.0 of the spec.

I was just going with the already used field names.

Just let me know whether to use the wrong-typed existing working "manufacture",
or the correct "manufacturer" instead? 
I don't have a preference.

**Comment by mrutkows** [comment]
> > Since this is a new field of component, should we use `manufacturer` instead of `manufacture`? I'd like to eventually change `metadata. manufacture` in v2.0 of the spec.
> 
> I was just going with the already used field names.
> 
> Just let me know whether to use the wrong-typed existing working "manufacture", or the correct "manufacturer" instead? I don't have a preference.

IMO, we should endeavor to correct to "manufacturer" where possible.

File: schema/bom-1.6.proto [hunk 2]
```diff
@@ -141,6 +141,8 @@ message Component {
   optional ComponentData data = 26;
   // Cryptographic assets have properties that uniquely define them and that make them actionable for further reasoning. As an example, it makes a difference if one knows the algorithm family (e.g. AES) or the specific variant or instantiation (e.g. AES-128-GCM). This is because the security level and the algorithm primitive (authenticated encryption) is only defined by the definition of the algorithm variant. The presence of a weak cryptographic algorithm like SHA1 vs. HMAC-SHA1 also makes a difference.
   optional CryptoProperties cryptoProperties = 27;
+  // The organization that manufactured the component (the "manufacturer", although the field is misspelled).
```
Comment (mrutkows) [inline schema/bom-1.6.proto:144]: I would recommend some qualification to the ack. of the misspelling (where it MUST be preserved for 1.x compatibility) to change from ", although the field is misspelled" to ", although the field name is acknowledged to misspelled in this version.".  To indicate/imply that it is left that way intentionally (for this version).

**Comment by jkowalleck** [comment]
i took the previous comemnts to heaert and will propose a significant change soon.
the idea kills two birds with one stone:
- correct the typo by adding a new field "$.metadata.manufacturer" (and deprecating "$.metadata.manufacture") & have consistent naming 
- solve the issue of #346

**Pull request closed without merging**